sanic/cli/inspector.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser
+
+from sanic.cli.base import SanicHelpFormatter, SanicSubParsersAction
+
+
+def make_inspector_parser(parser: ArgumentParser) -> None:
+    parser.add_argument("--host", "-H", default="localhost")
+    parser.add_argument("--port", "-p", default=6457, type=int)
+    parser.add_argument("--secure", "-s", action="store_true")
+    parser.add_argument(
+        "--raw",
+        action="store_true",
+        help="Whether to output the raw response information",
+    )
+
+    subparsers = parser.add_subparsers(
+        action=SanicSubParsersAction,
+        dest="action",
+        description=(
+            "Run one of the below subcommands. If you have created a custom "
+            "Inspector instance, then you can run custom commands.\nSee ___ "
```

Comment:
Yup, I do this intentionally since I will not write the docs and have a link until next week. Then I make a final PR before release with changelog info and fill in what is missing.